develop/codingguidelines/Attention/Minimum_DMA_version.md

```diff
@@ -8,3 +8,10 @@ Make sure the [MinimumRequiredVersion](xref:Protocol.Compliancies.MinimumRequire
 
 > [!TIP]
 > DIS performs a feature detection to detect whether the specified value for MinimumRequiredVersion is in accordance with the used features in the protocol.
+
+| Feature | Minimum DMA Version | Release Note |
+|----------|----------|----------|
+| Protocol clear and leave in FillArray    | 10.4.2.0-13691     | [Releas Note](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=38153)     |
```

Comment:
Could you link to the release note on docs instead? That way, this link will also be useful for non-Skyline users.